src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -288,7 +288,7 @@ def build_vm_msi_extension(vm_name, location, role_assignment_guid, port, is_lin
         'location': location,
         'dependsOn': [role_assignment_guid or 'Microsoft.Compute/virtualMachines/' + vm_name],
         'properties': {
-            'publisher': "Microsoft.ManagedIdentity",
+            'publisher': "Microsoft.ManagedIdentity.Edp",
```

Comment:
This and similar ones below must be reverted before merge. `.Edp` is the test version